website/blog/2023-05-18-ferretdb-meteor-mongodb-alternative.md

```diff
@@ -0,0 +1,89 @@
+To mitigate this, open-source alternatives like [FerretDB](https://www.ferretdb.io/) offer the opportunity to seamlessly replace MongoDB without compromising the commands and syntaxes you already know.
+
+In this article, we'll explore the possible synergy between FerretDB and Meteor.js and how you can build your applications without any concern for lock-in.
+
+## Brief History of Meteor.js
+
+Meteor.js is an open-source, all-in-one JavaScript framework, available publicly under the MIT license.
+Built with Node.js, Meteor.js comes with built-in MongoDB handlers, Blaze handlebars for efficient templating, a powerful pub/sub method for real-time data synchronization, and seamless integration with various Meteor.js plugins, and `npm` packages you need.
+
+This extensive ecosystem empowers developers to rapidly create quick cross-platform prototypes or large, complex applications with ease.
+With its open-source nature and availability under the MIT license, Meteor.js has gained popularity among not only mid-sized and enterprise companies like  [IKEA](https://www.ikea.com/), [Workpop](https://www.betterteam.com/workpop), [Accenture](https://www.accenture.com/us-en) but also individual developers and startups.
+
+The framework has been responsible for the development of remarkable apps, such as [Rocket.Chat](https://rocket.chat/), [Apify](https://apify.com/), and [Chatra](https://chatra.com/), among others.
+Its ability to efficiently handle real-time communication and collaboration make it an ideal platform for applications requiring instant updates and dynamic content.
+
+Meteor.js uses MongoDB as its standard database on the client, called Minimongo.
+However, [several concerns about MongoDB](https://blog.ferretdb.io/open-source-is-in-danger/) over the years have led some to seek (or demand) an open-source alternative that sufficiently replaces MongoDB without any hassles.
+
+Since MongoDB's license switch to SSPL from its former open-source license, there have been growing concerns about the nature of the license and how it may affect developers and companies building their applications with MongoDB.
+Several Meteor.js application developers are also in this corner.
+
+And this is what makes [the announcement of the General Availability version](https://blog.ferretdb.io/ferretdb-1-0-ga-opensource-mongodb-alternative/) of FerretDB – an open-source replacement to MongoDB – so exciting!
+
+## Introducing FerretDB
+
+FerretDB is the defacto open-source MongoDB alternative that converts MongoDB wire protocols with a backend on [PostgreSQL](https://www.postgresql.org/).
+Besides PostgreSQL, FerretDB also supports other database backends, such as [Tigris](https://www.tigrisdata.com/), [SAP Hana](https://www.sap.com/africa/products/technology-platform/hana.html), [SQLite](https://sqlite.org/), and many more to come.
```

Comment:
https://docs.ferretdb.io/understanding-ferretdb/#supported-backends is pretty clear that SQLite and Hana are not supported yet. They don't even work